
AN ERROR HAS OCCURRED [the index 'w' does not exist]

CALLSTACK
*FUNCTION [__main__()] testData/static_analyzer/w248_chain.nut line [7]

LOCALS
[a] 10
[x] TABLE
[vargv] ARRAY
[this] NULL
